docs/UserGuide.md

```diff
@@ -219,33 +292,78 @@ Format: `test INDEX o/OPTION`
 Examples:
 * `list` followed by `test 1 o/2` tests the 1st flashcard in the list with `OPTION 2` corresponding to the 2nd choice in the choices of the multiple choice question as the answer.
 
-### Displaying statistics for a Flashcard: `stats`
+### Displaying statistics for a Flashcard: `stats INDEX`
 
-Shows the Pie Chart for a specified Flashcard in the list.
+To view the statistics of a flashcard you can use the stats command.
 
-Statistics include:
+<div markdown="block" class="alert alert-info">
 
-* The number of times and the percentage the user answers the question associated with the flashcard correctly.
-* The number of times and the percentage the user answers the question associated with the flashcard incorrectly.
+:information_source: The INDEX refers to the the index number shown on the last displayed flashcard list and it <strong>must be a positive integer</strong>.
 
-Format: `stats INDEX`
+</div>
 
-* Displays the statistics of a flashcard at the specified `INDEX`.
-* The index refers to the index number shown in the displayed flashcard list.
-* The index **must be a positive integer** 1, 2, 3, …
+<div markdown="block" class="alert alert-dark">
```

Comment:
I can't really view this over GitHub but I think we might need to check this against what others have done for their sections cause I don't think anyone has used the `alert-dark` class yet.